applications/schedview-snapshot/templates/deployment.yaml

```diff
@@ -29,6 +29,7 @@ spec:
               drop:
                 - "all"
             readOnlyRootFilesystem: true
+          privileged: false
```

Comment:
This isn't a valid setting at this level of a container specification (it would go under `securityContext`, so I think Argo CD will either reject this manifest or will silently delete this setting. I'm not sure what this addition was intended to do, though. Containers are unprivileged by default so if this were under `securityContext` it would just be reasserting the default and would have no effect.

Reply:
When I didn't have it there, I was getting the following "event" from ArgoCD:

> policy disallow-privileged-containers/privileged-containers fail: validation error: Privileged mode is disallowed. The fields spec.containers[*].securityContext.privileged and spec.initContainers[*].securityContext.privileged must be unset or set to `false`. . rule privileged-containers failed at path /spec/

Originally, I didn't have a privileged keyword at all. When I added it as I did, it made the event go away. But, I did indeed miss the `securityContext` element in the complaint. I'm not clear on why what I did worked, or why it was necessary at all. I'll go ahead and move it under `securityContext` and check whether the complaint comes back.

Reply:
When I move it under `securityContext`, the event reappears.

Reply:
When I put it both next to and under `securityContext`, the event goes away again.

Reply:
When I take it out from the place where it should **not** go (next to, not under, `securityContext`), a second policy complaint also shows up:
`policy disallow-host-path/host-path fail: validation error: HostPath volumes are forbidden. The field spec.volumes[*].hostPath must be unset. . rule host-path failed at path /spec/`
I don't think I'm using hostpath volumes anywhere (and find/grep doesn't find any).
Maybe there's a bug in the policy configuration for usdf-inf somewhere?